src/shaders/conjurer_common.frag

```diff
@@ -52,10 +52,10 @@ float plot(vec2 st, float pct) {
 // Cartesian coordinates mean the usual x,y coordinates, where (0, 0) is at the center of the canopy
 // and (0.5, 0.5) is at the top right corner of the canopy.
 vec2 canopyToCartesianProjection(vec2 _st) {
-    float theta = _st.x * 2.0 * PI;
+    float theta = (_st.x - 0.5) * 2.0 * PI;
     // TODO: double check these numbers
     float r = _st.y * 0.88888888 + 0.111111111;
-    return vec2(r * cos(theta) * 0.5, r * sin(theta) * 0.5);
+    return vec2(r * cos(theta), r * sin(theta));
```

Comment:
Oh man, looks like my formula here was a little off 🤔 problem is, there are a few patterns/effects that rely on the old formula. Later today I might add back the old formula under a different name (`brokenCanopyToCartesianProjection`) for backwards compatibility and I will change them over to the correct formula over time.